charts/policies/allow-privilege-escalation-psp/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+# This is a Helm Chart Chart.yaml file. Used to build a Rancher Helm chart.
+---
+apiVersion: v2
+name: allow-privilege-escalation-psp
+description: Helm chart for deploying the allow-privilege-escalation-psp Kubewarden policy
+icon: https://www.kubewarden.io/images/icon-kubewarden.svg
+type: application
+home: https://www.kubewarden.io/
+kubeVersion: ">= 1.19.0"
+keywords:
+  - Security
+  - Infrastructure
+  - Monitoring
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.11
+
+appVersion: 0.1.11
+
+annotations:
+  # required ones:
+  catalog.cattle.io/certified: rancher # Any application we are adding as a helm chart
```

Comment:
missing default release name, namespace, kubeVersion, rancherVersion, display name, and permits os (different than os; this indicates whether your chart tolerates being deployed onto a windows cluster, while the os annotation indicates if you are deploying windows specific resources) annotations